GIAP.Server/Services/IIdentityProviderService.cs

```diff
@@ -17,9 +17,16 @@ public interface IIdentityProviderService
 
     /// <summary>
     /// Get an identity provider by its web url slug.
+    /// This method is used to find an IdentityProvider from a user input slug.
+    /// The user input slug is always converted to lowercase.
```

Comment:
This is an implementation detail (in the class, not in the interface).
Remove this comment